lib/cartopy/mpl/geoaxes.py

```diff
@@ -575,9 +575,8 @@ def _update_title_position(self, renderer):
     def __str__(self):
         return '< GeoAxes: %s >' % self.projection
 
-    def cla(self):
-        """Clear the current axes and adds boundary lines."""
-        result = super().cla()
+    def __clear(self):
```

Comment:
I don't think this can be the same as mpl's private version. So I _think_ you can go back to `_clear` here now that double  under is in MPL.

Reply:
Double underscore is class-private; it's mangled automatically to `_GeoAxes__clear`, so it won't clash with Matplotlib's (which is mangled to `_AxesBase__clear`).

Reply:
Ahh, yes of course. This looks good to me now.